todo-backend-typescript/README.adoc

```diff
@@ -15,51 +15,53 @@ Uses the `client-typescript-eventsource` library.
 
 The app is based on the Open API description stored in todo-api folder, however Open API code generators are not used in this case
 
-In addition, it demonstrates integration with Google Analytics.You will require to have valid GA Tracking ID, e.g. 'UA-XXXXXXXXX-X'.
+In addition, it demonstrates integration with Google Analytics.
 
-GA events:
+== Prerequisites
 
-`name: "todo-add", value: "10"`
+* Node @12 or higher
 
-`name: "todo-delete", value: "5"`
+* You need to setup a feature of type "Feature flag - boolean" in the FeatureHub Admin Console.
+Set feature key to "FEATURE_TITLE_TO_UPPERCASE".
 
-Once you launch the server, any call to "add" or "delete" to-do will generate a GA event accordingly.
+image::https://docs.featurehub.io/images/create-feature-title.png[Feature demo,800]
 
-Note: The events will appear in your GA console with ~12 hours delay.
+* You are required to have a Service Account created in the FeatureHub Admin Console with the "read" permissions for your desired environment.
+Once this is set, copy the SDK URL from the Service Accounts page for your desired environment and set it as an environment variable:
 
-More on GA integration can be found here https://docs.featurehub.io/#_google_analytics_integration
+You need to append at the front of it the edge url and it's port, like this `'http://[featurehub-edge-url]:[featurehub-edge-port]/features/[copied-sdk-url]`
 
-== Prerequisites 
+For example: `http://your-server:8553/features/default/4e439d38-1e87-4e77-84c3-2a32c112cfc5/Fpy5YpEfTeR5yQ15DUWEyvpQyfpjKHefMq9QhlodNzN8XH4by048X9Vjlj94fPieNc5nkagdfVoeAJxM`
 
-Node @12 or higher
+Note, the port will depend on which FeatureHub installation you are running.
+If you are running SSE by itself, or the Docker with "all-separate-postgres" it will default to 8553. If you are running Party Server, it will be 8903, if you are running Docker with All-In-One-h2 it will be 8085
 
-Follow readme instructions to build  `client-typescript-core` module
+Set SDK Url in your run configuration as:
 
-Follow readme instructions to build  `client-typescript-eventsource` module
+`process.env.FEATUREHUB_APP_ENV_URL`
```

Comment:
this isn't clear, i would say "set it as an environment variable, i.e

export FEATUREHUB_APP_ENV_URL=http://blah blah

Reply:
(mention: "see the run.sh file")